docs/source/getting_started/getting_started_hard_way.rst

```diff
@@ -0,0 +1,269 @@
++--------------------+----------+---------------------------------------------------------------------------------------------------------------------+
+| internal_port      | integer  | Port for communication between ordering service, YAC consensus and block loader for synchronization (default 10001) |
++--------------------+----------+---------------------------------------------------------------------------------------------------------------------+
+| pg_opt             | string   | Postgres credentials                                                                                                |
++--------------------+----------+---------------------------------------------------------------------------------------------------------------------+
+| redis_host         | string   | Redis host IP address                                                                                               |
++--------------------+----------+---------------------------------------------------------------------------------------------------------------------+
+| redis_port         | integer  | Port to access redis storage                                                                                        |
++--------------------+----------+---------------------------------------------------------------------------------------------------------------------+
+| max_proposal_size  | integer  | Maximum size of created proposals                                                                                   |
++--------------------+----------+---------------------------------------------------------------------------------------------------------------------+
+| proposal_delay     | integer  | The period of time (in ms) used to prepare proposal of transactions                                                 |
++--------------------+----------+---------------------------------------------------------------------------------------------------------------------+
+| vote_delay         | integer  | The period of time (in ms) of spreading vote across the network                                                     |
++--------------------+----------+---------------------------------------------------------------------------------------------------------------------+
+| load_delay         | integer  | The period of time (in ms) between synchronizations between peers                                                   |
++--------------------+----------+---------------------------------------------------------------------------------------------------------------------+
+
+Example:
+
+.. code:: json
+
+  {
+    "block_store_path" : "/tmp/block_store/",
+    "torii_port" : 50051,
+    "internal_port" : 50541,
+    "pg_opt" : "host=localhost port=5432 user=postgres password=mysecretpassword",
+    "redis_host" : "localhost",
+    "redis_port" : 6379,
+    "max_proposal_size" : 10,
```

Comment:
so, playing around means changing these values in order to find an optimal value for them. For now, you can just generally say in more details what each of it means, and how we can change them, but in the future we should expand this I think with more interesting insights